src/components/Stack/Stack.scss

```diff
@@ -3,7 +3,7 @@
     --ydb-stack-offset-x: 4px;
     --ydb-stack-offset-y: 4px;
     --ydb-stack-offset-x-hover: 4px;
-    --ydb-stack-offset-y-hover: 8px;
+    --ydb-stack-offset-y-hover: 6px;
```

Comment:
This offset determines how far away are VDisks  when hovered. Example of 16px offset:

<img width="929" alt="Screenshot 2024-12-19 at 13 52 22" src="https://github.com/user-attachments/assets/b0221fa8-af5f-4a76-8dfd-94568f704b60" />